### PR TITLE
Update 18-working-with-marketing.rst

### PR DESCRIPTION
**Pull request by redweezul**
Created summary for Chapter 18.

### CODE REVIEW

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
 
-[QUICK INTRO]
+Summary of Chapter
+------------------
+This chapter discussed how technical writers can & should work with the Marketing department. It raises the question of docs' role: informative, sales-driven, or both?
 
-Discussion Summary
-------------------
\ No newline at end of file
+Docs can serve as marketing material, though they should aim for educating the user first & act as a sales pitch second. Marketing may not like the direct nature of docs, but it is crucial to maintain honesty & directness.
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Docs can serve as marketing material, though they should aim for educating the user first and act as a sales pitch second. Marketing may not like the direct nature of docs, but it is crucial to maintain honesty and directness.
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 2]
```diff
@@ -1,8 +1,45 @@
...
+
+Personas are a key element to both Tech Writers and Marketing. These do not always align though, as Marketing personas target buyers (Management), whereas docs target users (end users/developers).
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Personas are a key element to both tech writers and marketing. These do not always align though, as marketing personas target buyers (management), whereas docs target users (end users/developers).
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 3]
```diff
@@ -1,8 +1,45 @@
...
+
+How can Marketing help Docs:
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
How can marketing help docs:
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 4]
```diff
@@ -1,8 +1,45 @@
...
+* Ensure the doc team knows marketing priorities and feature naming conventions
+* Share customer feedback and perspective
+* Understand the market players and how competitors document their products
+* Align on customer goals
+* Understand future goals/plans
+
+How Docs can Help Marketing:
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
How docs can help marketing:
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 5]
```diff
@@ -1,8 +1,45 @@
...
+* Share your knowledge of what users find helpful
+* Information you develop could inform marketing content
+* Your product knowledge can improve the relevance of Marketing content
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
* Your product knowledge can improve the relevance of marketing content
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 6]
```diff
@@ -1,8 +1,45 @@
...
+
+Get & stay in contact with Marketing. Working together helps tighten the link between your docs & marketing content, avoids confusion for the user, and can make for a sharper content experience from interest to implementation.
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Get and stay in contact with marketing. Working together helps tighten the link between your docs and marketing content, avoids confusion for the user, and can make for a sharper content experience from interest to implementation.
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 7]
```diff
@@ -1,8 +1,45 @@
...
+
+Key Takeaways from Discussion
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Key takeaways from discussion
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 8]
```diff
@@ -1,8 +1,45 @@
...
+-----------------------------
+A big challenge: push-back from Marketing to make content more sales-y.
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
A big challenge: push-back from marketing to make content more sales-y.
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 9]
```diff
@@ -1,8 +1,45 @@
...
+
+Both tech writers & marketing aim to help a user achieve a goal: Selling the product that enables the user to achieve said goal (Marketing), and educating the user on how to use the product in this manner (Tech Writer).
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Both tech writers and marketing aim to help a user achieve a goal: selling the product that enables the user to achieve said goal (marketing), and educating the user on how to use the product in this manner (tech writer).
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 10]
```diff
@@ -1,8 +1,45 @@
...
+Tech Writers can share the following with Marketing:
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Tech writers can share the following with marketing:
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 11]
```diff
@@ -1,8 +1,45 @@
...
+* Expertise on the product.
+* Tutorials based on Marketing's sales points.
+* Easy links to docs, so they can point users to the appropriate material
+
+Marketing can share the following with Tech Writers:
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Marketing can share the following with tech writers:
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 12]
```diff
@@ -1,8 +1,45 @@
...
+* Communications advice when you have to share bad news (forced plan changes/migrations, dropping a feature, etc.).
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
* Communications advice when you have to share bad news (such as forced plan changes/migrations or dropping a feature).
```

File: docs/book-club/splunk-product-docs/18-working-with-marketing.rst [hunk 13]
```diff
@@ -1,8 +1,45 @@
...
+* Assistance with ensuring the places that hold my content are "on brand."
+* Design tips for screenshots and templates.
+
+Explain features and benefits to them in ways that they find more understandable than when the developers or engineers do the explaining. 
+Also, point out when the marketing docs aren't technically accurate or are missing some major technical benefits.
+
+Use the same language to describe products, and how to use them. May require differences in tone between docs & marketing, but aim to keep the user experience as similar as possible.
```
Comment (StarfallProjects) [inline docs/book-club/splunk-product-docs/18-working-with-marketing.rst]:
```suggestion
Use the same language to describe products, and how to use them. May require differences in tone between docs and marketing, but aim to keep the user experience as similar as possible.
```

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Thanks for contributing :-)

Only changes are:
- making capitalization consistent
- removing a few "&" and "etc."

**Comment by redweezul** [comment]
No problem. I accepted all these suggestions. If I'm doing this right, I'll submit a new pull request?

**Comment by StarfallProjects** [comment]
@redweezul I think now they're all resolved we can just merge? But not sure if we need further approval from someone at WTD - @plaindocs ?

**Comment by plaindocs** [comment]
As you're merging into your feature branch there is no need for further approval.

**Pull request merged**